src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const k in robot) {
```

Comment:
It`s not clear what this k-variable means, change it to a clear one (checklist point 3).